Xabe.FFmpeg/FFmpegWrapper.cs

```diff
@@ -49,25 +51,49 @@ internal Task<bool> RunProcess(string args, CancellationToken cancellationToken)
                     // https://github.com/Microsoft/vs-threading/blob/master/doc/analyzers/VSTHRD101.md
                     var ctr = cancellationToken.Register(() =>
                     {
-                        process.StandardInput.Write("q");
-                        Task.Delay(1000 * 5).ConfigureAwait(false).GetAwaiter().GetResult();
-
-                        try
+                        if (Environment.OSVersion.Platform != PlatformID.Win32NT)
                         {
-                            if (!process.HasExited)
+                            try
                             {
-                                _wasKilled = true;
-                                process.Kill();
+                                process.StandardInput.Write("q");
+                                Task.Delay(1000 * 5).ConfigureAwait(false).GetAwaiter().GetResult();
+                            }
+                            catch (InvalidOperationException)
+                            {
+                            }
+                            finally
+                            {
+                                if (!process.HasExited)
+                                {
+                                    process.CloseMainWindow();
+                                    process.Kill();
+                                    _wasKilled = true;
+                                }
                             }
-                        }
-                        catch (InvalidOperationException)
-                        {
                         }
                     });
 
                     using (ctr)
                     {
-                        process.WaitForExit();
+                        using (var processEnded = new ManualResetEvent(false))
```

Comment:
RTSP_NotExistingStream_CancelledSucesfully stucks on windows cli dotnet test run without this change (both locally and appveyor)